packages/blockchain-wallet-v4-frontend/src/modals/Interest/WithdrawalForm/template.success.tsx

```diff
@@ -1,13 +1,12 @@
+import { Button, SpinningLoader, Text } from 'blockchain-info-components'
 import { compose } from 'redux'
 import { connect } from 'react-redux'
-import { FormattedMessage } from 'react-intl'
-import { InjectedFormProps, reduxForm } from 'redux-form'
-import React from 'react'
-
-import { Button, SpinningLoader, Text } from 'blockchain-info-components'
 import { convertBaseToStandard } from 'data/components/exchange/services'
 import { Exchange } from 'core'
 import { fiatToString, formatFiat } from 'core/exchange/currency'
+import { FormattedMessage } from 'react-intl'
+import { InjectedFormProps, reduxForm } from 'redux-form'
+import React from 'react'
```

Comment:
can you put the ordering of the imports back?

1st block is always 3rd party dependencies
2nd block is aliased imports
3rd block is relative path imports